packages/wallets/src/features/cashier/modules/Transfer/components/TransferFormAccountCard/__tests__/TransferFormAccountCard.spec.tsx

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
+import { APIProvider } from '@deriv/api-v2';
 import { render, screen } from '@testing-library/react';
+import WalletsAuthProvider from '../../../../../../../AuthProvider';
 import useDevice from '../../../../../../../hooks/useDevice';
```

Comment:
useDevice from hooks is already deprecated can you replace it with the one from deriv-com/ui

```suggestion
import { useDevice } from '@deriv-com/ui';
```